SmartDeviceLink/private/SDLStreamingVideoLifecycleManager.m

```diff
@@ -797,24 +1025,59 @@ - (void)sdl_sendVideoStartService {
     }
 }
 
-#pragma mark Setters / Getters
+- (void)sdl_applyVideoCapability:(SDLVideoStreamingCapability *)capability {
+    assert(nil != capability);
```

Comment:
```suggestion
    assert(capability != nil);
```
I'm not sure if asserting is a graceful way of handling the nil case here

Reply:
It does not handle nil but it fails early and besides nil checks are implemented in caller methods.